src/ImportDefinitionsBundle/Resources/public/pimcore/js/process_manager/export_contextmenu.js

```diff
@@ -0,0 +1,60 @@
+pimcore.registerNS("pimcore.plugin.importdefinitions.export.context_menu");
+
+pimcore.plugin.importdefinitions.export.context_menu = Class.create(pimcore.plugin.admin, {
+    getClassName: function() {
+        return "pimcore.plugin.importdefinitions.export.context_menu";
+    },
+
+    initialize: function() {
+        pimcore.plugin.broker.registerPlugin(this);
+    },
+
+    prepareObjectTreeContextMenu: function (tree, treeClass, menuItem) {
+        // TODO: replace with data store
+        var objectTypes = [
```

Comment:
in order to replace it with the actual data store, you have to populate the data-store before the right click happens, or async to the right click.